subiquity/server/controllers/filesystem.py

```diff
@@ -1453,7 +1453,8 @@ async def run_autoinstall_guided(self, layout):
         rp_input = layout.get("reset-partition", None)
         if rp_input:
             reset_partition = True
-            if isinstance(rp_input, (str, int)):
+            # bool is a sublcass of int
+            if isinstance(rp_input, str) or type(rp_input).__name__ == "int":
```

Comment:
Originally. I wanted to ask you to reverse the logic (i.e., starting with `if isinstance(rp_input, bool) ...  elif isinstance(rp_input, (str, int)`) but I'm not sure if other types besides bool were meant to be caught in the else block. So I'll let the choice up to you.

Reply:
s/sublcass/subclass/

Reply:
I supposed `type(rp_input) is int` should work as well, right?

Reply:
yeah I think `type(...) is <type>` feels more pythonic